paloalto-autofocus/info.json

```diff
@@ -1,180 +1,797 @@
+            "WF_SAMPLE": ""
+          },
+          "seenByDataSourceIds": [],
+          "summaryGenerationTs": "",
+          "whoisAdminCountry": "",
+          "whoisAdminEmail": "",
+          "whoisAdminName": "",
+          "whoisDomainCreationDate": "",
+          "whoisDomainExpireDate": "",
+          "whoisDomainUpdateDate": "",
+          "whoisRegistrant": "",
+          "whoisRegistrar": "",
+          "whoisRegistrarUrl": "",
+          "wildfireRelatedSampleVerdictCounts": {
+            "MALWARE": ""
+          }
+        },
+        "tags": [
+          {
+            "count": "",
+            "customer_name": "",
+            "description": "",
+            "doc_count": "",
+            "lasthit": "",
+            "public_tag_name": "",
+            "source": "",
+            "tag_class_id": "",
+            "tag_definition_id": "",
+            "tag_definition_scope": "",
+            "tag_definition_scope_id": "",
+            "tag_definition_status": "",
+            "tag_definition_status_id": "",
+            "tag_name": ""
+          }
+        ]
+      }
     },
     {
-      "operation": "hunt_url",
+      "operation": "get_url_reputation",
+      "title": "Get URL Reputation",
+      "description": "Retrieves a report from Palo Alto AutoFocus for the URL submitted to determine if it is suspicious based on the URL you have specified.",
       "category": "investigation",
-      "annotation": "hunt_url",
-      "description": "Hunt an URL and get list of associated tags from AutoFocus",
-      "title": "Hunt URL",
+      "annotation": "get_url_reputation",
+      "output_schema": {},
+      "enabled": true,
       "parameters": [
         {
           "title": "URL",
-          "name": "url",
+          "description": "Specify the URL for which you want to retrieve a Palo Alto AutoFocus report.",
+          "required": true,
+          "editable": true,
+          "visible": true,
+          "type": "text",
+          "name": "indicatorValue",
+          "tooltip": "Specify the URL for which you want to retrieve a Palo Alto AutoFocus report."
+        },
+        {
+          "title": "Include Tags",
+          "description": "Select the option to include or exclude AutoFocus tags. By default, this is set as True.",
+          "required": true,
+          "editable": true,
+          "visible": true,
+          "type": "checkbox",
+          "name": "includeTags",
+          "value": true,
+          "tooltip": "Select the option to include or exclude AutoFocus tags. By default, this is set as True."
+        }
+      ]
+    },
+    {
+      "operation": "get_file_reputation",
+      "title": "Get File Reputation",
+      "description": "Retrieves a report from Palo Alto AutoFocus for the file that you have submitted to determine if it is suspicious based on the file hash value you have specified.",
+      "category": "investigation",
+      "annotation": "get_file_reputation",
+      "output_schema": {},
```

Comment:
output schema is missing

Reply:
This is uncertified connector. Not present in the API documentation.